wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -119,7 +119,7 @@ final class SelfProfileViewController: UIViewController {
             apiVersion >= .v7 {
             self.teamMigrationBanner = SelfProfileViewCallToActionBannerHostingController(
                 actionCallback: { [weak self] action in
-                    self?.onTeamCreationBannerInteraction(action, apiVersion: apiVersion)
+                    self?.onTeamCreationBannerInteraction(action, apiVersion: .v7)
```

Comment:
Is hardcoding the API version correct?

Reply:
Forgot to remove it after debugging the UI. Nice catch. Thanks!